mscore/importmxmllogger.h

```diff
@@ -0,0 +1,36 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2017 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __IMPORTMXMLLOGGER_H__
+#define __IMPORTMXMLLOGGER_H__
+
+class QXmlStreamReader;
+
+namespace Ms {
+
+class MxmlLogger {
+public:
+      enum class Level : char {
+            TRACE, INFO, ERROR
```

Comment:
INFO and ERROR conflicts with something in a Windows specific header, so it doesn't build there, renaming to XINFO and XERROR fixes the build problem